fluent-package/yum/systemd-test/update-to-next-version.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/commonvar.sh
+
+# Install the current
+package="/vagrant/${distribution}/${DISTRIBUTION_VERSION}/x86_64/Packages/fluent-package-[0-9]*.rpm"
+sudo $DNF install -y $package
+sudo systemctl enable --now fluentd
+systemctl status --no-pager fluentd
+
+# Make a dummy pacakge for the next version
+case $distribution in
+    amazon)
+        sudo amazon-linux-extras install -y epel
+        ;;
+    *)
+        sudo $DNF install -y epel-release
+        ;;
+esac
+sudo $DNF install -y rpmrebuild
+# Example: "Release     : 1.el9"
+release_line=$(rpm -qip $package | grep "Release*:*")
+# Example: "1"
```

Comment:
how about using `rpmquery --queryformat="%{Release}"`

Reply:
Thanks!! I didn't know that command!! This is very useful!!
I fixed it to use the command!